job-frontend.rst

```diff
@@ -0,0 +1,55 @@
+:orphan:
+
+Job Posting: Frontend developer with design skills
+==================================================
+
+.. |role| replace:: frontend developer
+
+.. include:: /hiring/intro.rst
+
+About the job
+-------------
+
+This job will be our first |role| hire.
+We have a couple members of the team who have some knowledge in this space,
+but we will expect you to lead our efforts in this area.
+
+We have recently gotten a :doc:`grant to support scientific software</czi-grant-announcement>` in 2021.
+Given this grant, the job will have two major parts:
+
+* For 2021, around half your time will be spent working on work related to our grant.
+  This focuses on building a generic content embedding client in Javascript, and working to integrate it into Sphinx and other widely used platforms.
+* The other half of your time will be spent working on Read the Docs itself, outside of the grant funding.
+  We are working on a site redesign, along with many other places where we have low-hanging UX and design work.
+
+We have two major places that users interact with Read the Docs:
+
+* The "dashboard" where users manage their documentation builds.
```

Comment:
I referred to it as our application a few times, and usually use application vs documentation pages as nomenclature for the two.

```suggestion
* Our application, where users manage their documentation builds.
```